### PR TITLE
Added Cytron Maker Feather AIoT S3.

### PR DESCRIPTION
**Pull request by waiweng83**
Added support for new board.

### CODE REVIEW

File: ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+// If not defined neopixel will be use for flash writing instead
+#define LED_PIN               2
+#define LED_STATE_ON          1
+
+//--------------------------------------------------------------------+
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID                  0x303A
+#define USB_PID                  0x80FA
```
Comment (hathach) [inline ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h:67]: can you provide link to where this PID is allocated.

File: ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+#define USB_MANUFACTURER         "Cytron"
+#define USB_PRODUCT              "Maker Feather AIoT S3"
+
+#define UF2_PRODUCT_NAME         USB_MANUFACTURER " " USB_PRODUCT
+#define UF2_BOARD_ID             "Maker-Feather-AIoT-S3"
```
Comment (hathach) [inline ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h:72]: Board-ID must follow uf2 specs format, checkout other boards and uf2 spec for detail

**Review by hathach — CHANGES_REQUESTED** [review]
board-id must follow specs

File: ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h (hunk 1 above)
Comment (waiweng83) [reply ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h:67]: The PID is provided by espressif.
https://github.com/espressif/usb-pids/blob/main/allocated-pids.txt

File: ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h (hunk 2 above)
Comment (waiweng83) [reply ports/espressif/boards/cytron_maker_feather_aiot_s3/board.h]: Updated.

Thanks.

**Review by hathach — APPROVED** [review]
perfect, thank you

**Comment by waiweng83** [comment]
@hathach May I know how often a new release will be created?

**Comment by hathach** [comment]
we made release when needed (or requested for new boards) or there is major code changes. I will made a release soon enough.

**Comment by waiweng83** [comment]
> we made release when needed (or requested for new boards) or there is major code changes. I will made a release soon enough.

Ok noted. Thanks.

**Pull request merged**